src/elements/content-uploader/UploadInput.js

```diff
@@ -15,17 +15,27 @@ type Props = {
 };
 
 const UploadInput = ({
-    isMultiple = true,
-    isFolderUpload = false,
-    inputLabelClass = '',
-    inputLabel,
     handleChange,
-}: Props) =>
-    inputLabel ? (
-        // eslint-disable-next-line jsx-a11y/label-has-for
-        <label className={inputLabelClass}>
+    inputLabel,
+    inputLabelClass = '',
+    isFolderUpload = false,
+    isMultiple = true,
+}: Props) => {
+    const inputRef = React.useRef(null);
+
+    const onKeyDown = e => {
+        if (e.key === 'Enter' || e.key === ' ') {
+            if (inputRef.current) {
+                inputRef.current.click();
+            }
```

Comment:
Flow is not compatible with optional chaining `inputRef.current?.click()` so this will have to be used until we update Flow

Reply:
Also, I think it's more readable to have this as a separate if statement, rather than doing
`if ((e.key === 'Enter || e.key === ' ') && inputRef.current) { ... }`